src/chainbin/nnet3-chain-get-egs.cc

```diff
@@ -38,12 +38,49 @@ namespace nnet3 {
    you should do it later with nnet3-chain-normalize-egs.
 */
 
```

Comment:
Looks like you have inserted a function between the documentation above and the function it was intended for.  A couple lines of documentation for GetIvectorsForEgs would be helpful too.